packages/teleport/src/components/TabIcon/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Gravitational, Inc.
```

Comment:
2022 here as well?

Reply:
it was only the file name that changed, opting to leave as is